### PR TITLE
[reproducer] allow customize dns server via cifmw_reproducer_dns_servers

### PR DESCRIPTION
**Pull request by stuggi**
As a pull request owner and reviewers, I checked that:
- [X] Appropriate testing is done and actually running

### CODE REVIEW

File: ci_framework/roles/reproducer/defaults/main.yml [hunk 1]
```diff
@@ -27,3 +27,6 @@ cifmw_reproducer_crc_private_nic: enp2s0
 cifmw_reproducer_kubecfg: "{{ cifmw_libvirt_manager_configuration.vms.crc.image_local_dir }}/kubeconfig"
 cifmw_reproducer_params: {}
 cifmw_reproducer_run_job: true
+cifmw_reproducer_dns_servers:
```
Comment (cjeanner) [inline ci_framework/roles/reproducer/defaults/main.yml:30]: Please add this to the README.md

Comment (stuggi) [reply ci_framework/roles/reproducer/defaults/main.yml:30]: done

File: ci_framework/roles/reproducer/README.md [hunk 2]
```diff
@@ -16,6 +16,7 @@ None
 * `cifmw_reproducer_params`: (Dict) Specific parameters you want to pass to the reproducer. Defaults to `{}`.
 * `cifmw_reproducer_private_nic`: (String) Private NIC for compute and controller. Defaults to `eth1`.
 * `cifmw_reproducer_crc_private_nic`: (String) Private NIC for CRC node. Defaults to `enp2s0`.
+* `cifmw_reproducer_dns_servers`: List if dns servers which should be used by the CRC VM as upstream dns servers. Defaults to 1.1.1.1, 8.8.8.8.
```
Comment (pablintino) [inline ci_framework/roles/reproducer/README.md:19]: Small typo `of`

Comment (dasm) [reply ci_framework/roles/reproducer/README.md:19]: :+1:

**Review by dasm — APPROVED** [review]
/lgtm

File: ci_framework/roles/reproducer/README.md (hunk 2 above)
Comment (stuggi) [reply ci_framework/roles/reproducer/README.md]: thanks, fixed

**Pull request merged**